src/core.js

```diff
@@ -340,6 +340,19 @@ define([
      *              - questions: [mugType, ...]
      */
     fn.getQuestionGroups = function () {
+        let mediaGroup = {
```

Comment:
Unless I'm misunderstanding it, should `mediaGroup` not be injected into the object being returned? (and replace the similar one already in there)

Reply:
Good catch! I was porting these changes over from HQ code (as I was testing there) and this must have slipped through. This has been fixed (77bd00a1a7cb01de518011f3339e052f30e3bf65)